fetch.bs

```diff
@@ -9062,6 +9062,107 @@ done only by navigations). The <a>fetch controller</a> is also used to
+
+<p>The Fetch standard enforces some of the mechanisms that constitute the Same Origin Policy (SOP),
+which is a critical aspect of web security. Specifically, Fetch imposes SOP restrictions on network
+access.
+
+<p>In a nutshell, when a user accesses a certain origin using their browser or other user agent,
+they don't expect that the user agent implicitly grants that <a for=/>origin</a> full access to
+their network, which could, for example, be an intranet, despite the user agent itself having that
+access.
+
+<p>In addition, the user agent may include footprint that identifies the user to the origin,
+explicitly using cookies or client certificates, or implicitly using, for example, the user-agent's
+IP address.
+
+<p>To keep this information from leaking, SOP restricts requests that come from the visited
+<a for=/>origin</a> to only target that same origin, while allowing a few mechanisms for an origin
+to access cross-origin data safely.
+
+<h4 id=fetch-elsewhere-cors>Cross-origin resource sharing (CORS)</h4>
+
+<p>The main way to access cross-origin data is via the <a>cors protocol</a>, described in detail in
+this standard. CORS (Cross Origin Resource Sharing) is built on the principle that origins that want
+to access data from other origins need to identify themselves, and get explicit approval from the
+server to access that data, thus not relying on any implict trust given to the user agent or to the
+machine the user agent is installed on.
+
+<p>This is done by sending the `<a http-header><code>Origin</code></a>` header with CORS-enabled
+requests, and expecting a matching `<a http-header><code>Access-Control-Allow-Origin</code></a>`
+response header in return.
+
+<p>The <a>CORS-preflight fetch</a> mechanism extends this further, by sending that identification
+handshake as a separate request, prior to sending the payload of the request itself.
+
+<p>For any new spec invoking <a for=/>fetch</a> to access cross-origin rersources, CORS should be
+the only option. This has been implemented in multiple newer standards such as web fonts and module
+scripts.
+
+<h4 id=fetch-elsewhere-no-cors>No-CORS requests</h4>
+
+<p>Embedding cross-origin resources, without the ability to read them, existed prior to CORS.
+Cross-origin images, scripts, stylesheets & media are embeddable in a <a for=/>document</a> without
+the embedding <a for=/>origin</a> identifying themselves and receive explicit confirmation.
+
+<p>To maintain backwards compatibility, this form of fetching is still supported. Instead of
+restricting the network access, the embedder is restricted in what they are allowed to do with the
+<a for=/>response</a>. They can embed it in their document and present it to the user, but this has
+to be done in a way that's opaque to the embedding origin. Only the user should have access to the
+resource, not the embedder.
+
+<p>This mechanism of fetching should not be used in new specs. However, specs should be written with
```

Comment:
Mention that it's the default request mode, and so every new spec should set a request mode explicitly.